dali/test/python/test_pipeline_multichannel.py

```diff
@@ -0,0 +1,274 @@
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nvidia.dali.pipeline import Pipeline
+import nvidia.dali.ops as ops
+import nvidia.dali.types as types
+import nvidia.dali as dali
+from nvidia.dali.backend_impl import TensorListGPU
+import numpy as np
+from numpy.testing import assert_array_equal, assert_allclose
+import os
+from test_utils import check_batch
+from test_utils import compare_pipelines
+from test_utils import RandomDataIterator
+import cv2
+
+test_data_root = os.environ['DALI_EXTRA_PATH']
+multichannel_tiff_root = os.path.join(test_data_root, 'db', 'single', 'tiff', 'multichannel')
+
+def crop_func_help(image, layout, crop_y = 0.2, crop_x = 0.3, crop_h = 220, crop_w = 224):
+    if layout == types.NFHWC:
+        assert len(image.shape) == 4
+        H = image.shape[1]
+        W = image.shape[2]
+    elif layout == types.NHWC:
+        assert len(image.shape) == 3
+        H = image.shape[0]
+        W = image.shape[1]
+
+    assert H >= crop_h
+    assert W >= crop_w
+
+    start_y = int(np.round(np.float32(crop_y) * np.float32(H - crop_h)))
```

Comment:
Please update according to https://github.com/NVIDIA/DALI/blob/master/dali/test/python/test_operator_crop_mirror_normalize.py#L158.  As https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.around.html#numpy.around states that:
```
For values exactly halfway between rounded decimal values, NumPy rounds to the nearest even value. Thus 1.5 and 2.5 round to 2.0, -0.5 and 0.5 round to 0.0, etc. 
```
And our C++ code just rounds to the nearest.

Reply:
done